pkg/chains/formats/slsa/v2alpha2/internal/taskrun/taskrun.go

```diff
@@ -91,6 +91,15 @@ func internalParameters(tro *objects.TaskRunObject) map[string]any {
 // externalParameters adds the task run spec
 func externalParameters(tro *objects.TaskRunObject) map[string]any {
 	externalParams := make(map[string]any)
+	// add top level task config
+	if t := tro.Status.Provenance; t != nil && t.RefSource != nil && tro.Spec.TaskRef != nil {
+		buildConfigSource := map[string]any{
+			"ref":        t.RefSource.Digest,
```

Comment:
Same comments for pipeline level apply here.

Reply:
done.